it/controllers/testSupport/ITCoreTestData.scala

```diff
@@ -84,5 +84,16 @@ trait ITCoreTestData extends TryValues {
   def addASmallProducerFullAnswers = addASmallProducerPartialAnswers.success.value
       .set(AddASmallProducerPage, AddASmallProducer(Some("Super Cola Ltd"),"XZSDIL000000234",1000L, 1000L))
 
+  def smallProducerDetaisPartialAnswers = addASmallProducerFullAnswers.success.value
+    .set(AddASmallProducerPage, AddASmallProducer(Some("Super Cola Ltd"),"XZSDIL000000234",1000L, 1000L))
```

Comment:
Best create a val for those hard coded values and reuse them instead of repeating the same hard coded values in two tests.